zerver/lib/stream_recipient.py

```diff
@@ -37,24 +38,20 @@ def populate_for_recipient_ids(self, recipient_ids: List[int]) -> None:
             return
 
         # see comment at the top of the class
-        id_list = ', '.join(str(recip_id) for recip_id in recipient_ids)
-        query = '''
+        query = SQL('''
             SELECT
                 zerver_recipient.id as recipient_id,
                 zerver_recipient.type_id as stream_id
             FROM
                 zerver_recipient
             WHERE
-                zerver_recipient.type = %d
+                zerver_recipient.type = %s
             AND
-                zerver_recipient.id in (%s)
-            ''' % (Recipient.STREAM, id_list)
+                zerver_recipient.id in %s
+        ''')
 
-        self._process_query(query)
-
-    def _process_query(self, query: str) -> None:
         cursor = connection.cursor()
-        cursor.execute(query)
+        cursor.execute(query, (Recipient.STREAM, tuple(recipient_ids)))
```

Comment:
Should aim to use the named parameters + dictionary format; while we're cleaning this up?  It's so much more readable, especially with multiple parameters like here.

Reply:
Sure why not, done.